### PR TITLE
nix: run all operations with "--option extra-experimental-features 'n…

### PR DESCRIPTION
**Pull request by Artturin**
…iterals'"

https://github.com/nixos/nixpkgs/commit/92b4f173803f65531e066321934e7d1ee7eb5090

```
error: URL literals are disabled

       at /home/artturin/nixpkgs/.worktree/1/pkgs/games/tennix/default.nix:8:11:

            7|   src = fetchgit {
            8|     url = git://repo.or.cz/tennix.git;
             |           ^
            9|     rev = "refs/tags/tennix-${version}";
nix-env -qa --out-path --no-name -f outpaths.nix --arg checkMeta false      215.81s user 8.50s system 99% cpu 3:44.39 total
1 artturin@nixos ~/nixpkgs/.worktree/1 (git)-[master~1] % nix-env -qa --out-path  --no-name  -f outpaths.nix --arg checkMeta false --verbose--option extra-experimental-features 'no-url-literals'
```

### CODE REVIEW

**Comment by vcunat** [comment]
Example where this feature would've been desirable: https://github.com/NixOS/nixpkgs/pull/204859

**Comment by SuperSandro2000** [comment]
As the person that merged that PR: Why is ofborg not already complaining about this!?

**Comment by vcunat** [comment]
Most likely it just hasn't been noticed/reported.

File: ofborg/src/nix.rs [hunk 1]
```diff
@@ -68,13 +68,16 @@ impl Operation {
...
                 ]);
             }
             Operation::NoOp { ref operation } => {
                 operation.args(command);
             }
             Operation::Evaluate => {
-                command.args(&["--eval", "--strict", "--json"]);
+                command.args(&["--eval", "--strict", "--json", "--option", "experimental-features", "no-url-literals"]);
             }
             _ => (),
```
Comment (cole-h) [inline ofborg/src/nix.rs:105]: Looks like we probably want a new arm for `Operation::Instantiate` -- that operation is used for the `nixos` and `nixos-options` (and others) checks.

**Review by cole-h — DISMISSED** [review]

**Review by cole-h — APPROVED** [review]
Let's give it  a shot.

**Comment by cole-h** [comment]
https://github.com/NixOS/nixpkgs/pull/207318

ofborg catches url literals now.

**Pull request merged**